### DISCUSSION TITLE
How to Add multiple Docker server into my Coder

### DISCUSSION THREAD

**Discussion by sjkeerthi** [coder/coder#13573] [General] [answered]
Hi I have installed coder on the same box I installed docker now I able to provision docker container to each developer 
Now the base machine is less now I plan to create another VM and point that docker to be added into my existing coder 
server since I cannot provider each every developer two different coder url server 

Assume If I have setup a docker with remote port enable how to attach to the existing coder server so that the developer can 
create there containers under the same one coder url

**Comment by matifali** [comment, marked as answer]
Please have a look at https://coder.com/docs/@v2.13.5/platforms/docker#using-remote-docker-host to learn how you can link remote docker hosts with Coder.

**Comment by sjkeerthi** [comment]
As per your link I could not able to see any doc for using-remote-docker-host
![image](https://github.com/user-attachments/assets/c12c648d-39aa-4cc5-b6c9-bc3eba2f7e19)

**Reply by matifali** [reply]
I updated the link to route to the correct version of the docs. We have refactored our docs, which caused these instructions to be missed.
To fix I opened #15494

**Comment by sjkeerthi** [comment]
When I choose Docker Template I could not see to point my docker remote IP address 
How do I specify  those.
